.github/workflows/example-testing.yml

```diff
@@ -2,61 +2,51 @@
 #   Bash shells do not use ~/.profile or ~/.bashrc so these shells need to be explicitely declared as shell: bash -el {0}
 #   https://github.com/marketplace/actions/setup-miniconda#important
 #
-name: Example Tests
+name: Examples
 
 on: [push, pull_request]
 
 jobs:
-  examples:
+  tests:
     runs-on: ubuntu-latest
     defaults:
       run:
-        shell: bash -el {0}
+        shell: bash -l {0}
     container:
-      image: jammont/isofit:latest
-      env:
-        ENV_NAME: test
+      image: jammont/isofit:amd64
+      options: --user root
 
     steps:
+    - name: Debugging
+      run: |
+        du --max-depth=1 -h /__w/
+        df -h
+
     - name: Checkout
       uses: actions/checkout@v2
 
     - name: Install Isofit
       run: |
-        source /root/.bashrc
-        pip install --editable .
-
-    # - name: Running Example - image_cube/small
-    #   run: |
-    #     cd examples/image_cube
-    #     bash run_small.sh
-    #
-    # - name: Running Example - image_cube/medium
-    #   run: |
-    #     cd examples/image_cube
-    #     bash run_medium.sh
-
-    - name: Running Example - 20151026_SantaMonica
+        echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
```

Comment:
This `echo` needs to happen in a step before any `python` or `pip` calls are made, ie:
```yaml
- name: Enable ISOFIT Environment
      run: |
        echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
- name: Instal ISOFIT
      run: |
        pip install ...
```